leasing/tests/api/test_create_lease.py

```diff
@@ -10,8 +10,8 @@
 from leasing.models import Lease
 
 
-@pytest.mark.django_db
-def test_create_lease(django_db_setup, admin_client, contact_factory):
+@pytest.fixture
+def lease_create_data(contact_factory):
```

Comment:
This fixture could be moved to the conftest where all of the other fixtures are. The name could also be more descriptive. Like "lease_data_dict_with_contacts" or something.